From 22e2451f25a5f1418758e4735e86f144fabf9ee3 Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Tue, 31 Jul 2012 13:44:08 +0300
Subject: [PATCH 271/311] confc.c: don't use c2_rpc_client_{init,fini}()

c2_rpc_client_{init,fini}() belong RPC UT and may not be used by confc.

LogD #599
RB: r/939/diff/1/?file=25263#file25263line1095
---
 conf/confc.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/conf/confc.c b/conf/confc.c
index 0756b91..b3d7f24 100644
--- a/conf/confc.c
+++ b/conf/confc.c
@@ -1062,9 +1062,9 @@ static int confd_connect(struct c2_rpc_client_ctx *rpc, const char *confd_addr)
 	rpc->rcx_remote_addr = confd_addr;
 	XXX; /* set other fields of `rpc' */
 
-	rc = c2_rpc_client_init(rpc);
+	rc = c2_rpc_client_start(rpc);
 	if (rc != 0) {
-		(void) c2_rpc_client_fini(rpc);
+		(void) c2_rpc_client_stop(rpc);
 		rpc->rcx_remote_addr = NULL;
 	}
 	return rc;
@@ -1076,7 +1076,7 @@ static void confd_disconnect(struct c2_rpc_client_ctx *rpc)
 
 	C2_PRE(rpc->rcx_remote_addr != NULL);
 
-	rc = c2_rpc_client_fini(rpc);
+	rc = c2_rpc_client_stop(rpc);
 	if (rc != 0)
 		C2_ADDB(report error);
 }
-- 
1.8.3.2

